### PR TITLE
Bait tally preliminary

### PR DESCRIPTION
**Pull request by Vaylann**
A preview of sorts?

Some notes:
- doesn't use templates yet
- fish train's bait list remains unchanged
- there's also some inline css for the bait button - I'm not sure where to put that

### CODE REVIEW

**Comment by icykoneko** [comment]
I do plan to review this - just very preoccupied at the moment, sorry.

File: js/app/viewmodel.js [hunk 1]
```diff
@@ -468,6 +468,65 @@ let ViewModel = new class {
...
     });
 
+    $('#bait-button').on('click', function(e) {
+		//Helper method	for tallying fishes and their intuition fish
+		function mapFish(fish, map) {
+			if (fish.bait.length > 0) {
+				let bait = fish.bait[0];
+				
+				if (!map.has(bait.id)) {
+					const fishMap = new Map();
```
Comment (icykoneko) [inline js/app/viewmodel.js:478]: This line doesn't appear to do anything since line 481 is going to replace it.

**Comment by Vaylann** [comment]
Thanks!

Also, I've started looking at moving it into a template like fishguide. No guarantees I'll be able to manage though...

**Comment by Vaylann** [comment]
Done...

**Review by icykoneko — APPROVED** [review]
Outstanding work!  Thank you very much for implementing this great new feature!

**Pull request merged**